azure-toolkit-libs/azure-toolkit-eventhubs-lib/src/main/java/com/microsoft/azure/toolkit/lib/eventhubs/EventHubsInstance.java

```diff
@@ -69,6 +78,55 @@ public void disableSending() {
         updateStatus(EntityStatus.SEND_DISABLED);
     }
 
+    public boolean isListening() {
+        return Objects.nonNull(this.consumerAsyncClient);
+    }
+
+    public void publishEvents(String message) {
+        final IAzureMessager messager = AzureMessager.getMessager();
+        try (final EventHubProducerClient producer = new EventHubClientBuilder()
+                .connectionString(getOrCreateConnectionString(Collections.singletonList(AccessRights.SEND)))
+                .buildProducerClient()) {
+            EventDataBatch eventDataBatch = producer.createBatch();
+            final EventData eventData =  new EventData(message);
+            if (!eventDataBatch.tryAdd(eventData)) {
+                producer.send(eventDataBatch);
+                eventDataBatch = producer.createBatch();
+                if (!eventDataBatch.tryAdd(eventData)) {
+                    throw new AzureToolkitRuntimeException("Event is too large for an empty batch. Max size: "
```

Comment:
is there any other case which will break `tryAdd`? 
If so, is there any way to get the error message?
If not, do we need to add validation for message size before publishing Events?

Reply:
Use the example code from azure docs.
https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-java-get-started-send?tabs=passwordless%2Croles-azure-portal#write-code-to-send-messages-to-the-event-hub
Same example in azure sdk github
https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/eventhubs/azure-messaging-eventhubs#create-an-event-hub-producer-and-publish-events